spec/forms/provider_interface/edit_user_permissions_wizard_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+RSpec.describe ProviderInterface::EditUserPermissionsWizard do
+  describe '.from_model' do
+    let(:provider_permissions) { build_stubbed(:provider_permissions, make_decisions: true, view_safeguarding_information: true) }
+    let(:store_value) { {} }
+    let(:store) { instance_double(WizardStateStores::RedisStore, read: store_value) }
+    let(:wizard) { described_class.from_model(store, provider_permissions) }
```

Comment:
```suggestion
subject(:wizard) { described_class.from_model(store, provider_permissions) }
```

Reply:
Is there an advantage to doing this? I'm happy to change it, but we aren't really using the subject capability here

Reply:
I think its just convention for the thing you are interested in expecting upon to be the subject, makes it immediately clear to the person looking at the code that this is what we care about. It doesnt matter at all for functionality here though, so its ok to leave it.